tests/app.test.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../client/src/components/App';
+
+it('should have text', () => {
```

Comment:
@Lamanova this is a mock test. Please remove before committing. Also do you have any tests for this code that you want to add?